packages/react-component-library/src/components/Timeline/timeline_scales.ts

```diff
@@ -0,0 +1,112 @@
+import {
```

Comment:
Should this module be in the `hooks` directory and following custom hook convention?

Reply:
I appreciate there is another `useTimelineScale` hook in there that is used to manipulate the store.

This module seems to be related to initialisation?

Reply:
There was originally the mindset that the consumer could provide their own scale options although that feels like it might not be likely and maybe they will just provide min/max option. The initialisation of scale options has been moved to `initialiseState` as that feels cleaner with scope to change this in future.